aws/v12.2.0/release.diff

```diff
@@ -0,0 +1,66 @@
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v12.1.3                                                 |    name: v12.2.0
```

Comment:
12.1.4 has been merged. Would you like me to updated this release definition or prefer to do it yourself?

Please don't downgrade external-dns to 1.3.0

Reply:
I was aware of AWS v12.1.4 and I just merged master, updated release.yaml and regenerated release.diff to take into account v12.1.4.